x-pack/plugins/search_connectors/common/constants.ts

```diff
@@ -8,7 +8,7 @@
 import { ConnectorClientSideDefinition } from './types';
 import { docLinks } from './doc_links';
 
-export const CONNECTORS_DICT: Record<string, ConnectorClientSideDefinition> = {
+export const getConnectorsDict = (): Record<string, ConnectorClientSideDefinition> => ({
```

Comment:
I'd love a comment here on what led us to structure it like this :)

Reply:
okok will add!